<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>punto3</class>
 <widget class="QDialog" name="punto3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1229</width>
    <height>829</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="imgM5">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/monedas/moneda5.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lbnCantidadTtotal">
       <property name="text">
        <string>Cantidad:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lbMon100Dev">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lbMon500Dev">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btn200mas">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/boton/mas.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="4" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btn100mas">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/boton/mas.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLabel" name="lbCmoneda100">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="lbn500">
       <property name="text">
        <string>500 Pesos</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="imgMon500">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/monedas/moneda5.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="4" colspan="2">
      <widget class="QLineEdit" name="txtCantidadTotal">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="lbCmoneda500">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="imgM2">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/monedas/moneda2.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="imgDevolver">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/monedas/refund.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="lbn100">
       <property name="text">
        <string>100 pesos</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="imgMon200">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/monedas/moneda2.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="3">
      <widget class="QLabel" name="lbnCantidadDevuelta">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Su devuelta es de:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLabel" name="lbCmoneda200">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btn500mas">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/boton/mas.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lbMon200Dev">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="5" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btn100menos">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/boton/boton-menos.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="5" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btn200menos">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/boton/boton-menos.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="imgM1">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/monedas/moneda1.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="imgMon100">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/monedas/moneda1.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="3">
      <widget class="QLabel" name="imgVM">
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>200</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/imgVD/vendigMachine.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="lbn200">
       <property name="text">
        <string>200 pesos</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5" alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btn500menos">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border-image: url(:/boton/boton-menos.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3" alignment="Qt::AlignHCenter">
      <widget class="QLineEdit" name="txtTotalDev">
       <property name="text">
        <string>0</string>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="lbnMID">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>50</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>316</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Vending Machine&lt;/span&gt;&lt;br/&gt;Autómata de estados finitos determinísticos&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="0">
      <widget class="QLabel" name="LbVM">
       <property name="text">
        <string>VM= {s,Σ,δ,i,f}</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="LbnLogica">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>129</height>
        </size>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Definición formal de la Vending Machine&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Autómata de estados finitos determinístico, que resuelva el problema de recibir 1300 pesos en una máquina expendendedora que recibe solo monedas de 500, 200 y 100.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" alignment="Qt::AlignHCenter">
      <widget class="QPlainTextEdit" name="txtDefVM">
       <property name="minimumSize">
        <size>
         <width>900</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>150</height>
        </size>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Files/img/imgVM.qrc"/>
 </resources>
 <connections/>
</ui>
